client/js/upload-handler/xhr.upload.handler.js

```diff
@@ -118,7 +118,8 @@ qq.XhrUploadHandler = function(spec) {
                     name: uploadData.name,
                     remaining: uploadData.chunking.remaining,
                     size: uploadData.size,
-                    uuid: uploadData.uuid
+                    uuid: uploadData.uuid,
+                    loaded: uploadData.loaded
```

Comment:
This is not necessary. The number of bytes remaining is already something we can calculate based on the chunk size and the remaining chunks. 

Also, please do not combine unrelated changes into a single pull request. I'd very much like to keep changes simple and focused.